src/Dom/Document.php

```diff
@@ -1021,6 +1023,34 @@ private function restore_doctype_node( $html ) {
 		return preg_replace( self::HTML_RESTORE_DOCTYPE_PATTERN, '\1!\3\4>', $html, 1 );
 	}
 
+	/**
+	 * Normalizes HTML attributes to be HTML5 compatible.
+	 *
+	 * Conditionally removes html[xmlns], and converts html[xml:lang] to html[lang].
+	 */
+	private function normalize_html_attributes() {
+		$html = $this->getElementsByTagName( 'html' )->item( 0 );
```

Comment:
```suggestion
		$html = $this->documentElement;
```

Reply:
Ah, good point. I didn't think about that.

Reply:
Committed in https://github.com/ampproject/amp-wp/pull/4194/commits/7aaf8fd016967e130c9564bdeed9f0c9aee944f7